packages/runner-ct/src/app/app.tsx

```diff
@@ -16,6 +17,7 @@ import { ReporterHeader } from './ReporterHeader'
 import EventManager from '../lib/event-manager'
 import { Hidden } from '../lib/Hidden'
 import { SpecList } from '../SpecList'
+import { findDOMNode } from 'react-dom'
```

Comment:
This api is deprecated. Use refs instead

Reply:
done